smartgymapi/lib/validation/sport_scheme.py

```diff
@@ -1,8 +1,26 @@
-from marshmallow import Schema, fields
+from dateutil.parser import parse
+
+from marshmallow import Schema, fields, ValidationError, validates
 
 
 class SportScheduleSchema(Schema):
     id = fields.Str(dump_only=True)
-    user_id = fields.UUID(required='User id is required')
+    user_id = fields.UUID(dump_only=True)
     name = fields.Str(required='Name is required')
     reminder_minutes = fields.Integer(required='Reminder is required')
+    time = fields.Str(required='Time is required')
+    weekdays = fields.List(fields.Integer(), required='Weekday is required')
+    is_active = fields.Boolean(default=True)
+
+    @validates('time')
+    def validate_time(self, time_):
+        try:
+            parse(time_)
+        except:
+            raise ValidationError(time_ + ' Is not a valid time')
+
+    @validates('weekdays')
+    def validate_weekday(self, weekdays):
+        for weekday in weekdays:
+            if weekday > 7:
+                raise ValidationError('Weekdays are not valid')
```

Comment:
"Weekdays can only contain a value ranging from 1 to 7". Overigens klopt je validatie niet want ik kan nu -1 invullen. Zie `class marshmallow.validate.Range(min=None, max=None, error=None)` http://marshmallow.readthedocs.io/en/latest/api_reference.html#range